.env

```diff
@@ -0,0 +1,2 @@
+VITE_STRIPE_PUBLIC_KEY='pk_test_51Nhf4NLtxhLqEU8X8yobOruIFWfeBEwp5xQF7OEuVx2pSGMUWzfQ8LLzPYU4UZoOiDCv0v096VLwffvMLLGQL0hQ005wLyXDmR'
```

Comment:
env should be in gitignore

Reply:
fixed in [fix: added env to gitignore file and stop tracking it](https://github.com/ZenBit-Tech/faq_team_fe/pull/11/commits/46a2243175a98b53a959f68792e11bc9fd85f26f)